model_compression_toolkit/core/runner.py

```diff
@@ -83,6 +84,8 @@ def core_runner(in_model: Any,
                                               the attached framework operator's information.
         target_kpi: KPI to constraint the search of the mixed-precision configuration for the model.
         tb_w: TensorboardWriter object for logging
+        model_leaf_layers (list): List of the module's custom layers, these layers shouldn't be divided into
```

Comment:
List of the module's custom layers. These layers shouldn't be divided into their submodules. Please note that their quantization will not be optimized.